wce/wce_wld_raw.go

```diff
@@ -519,22 +482,34 @@ func (wce *Wce) WriteWldRaw(w io.Writer) error {
 
 	}
 
+	// Write non-spell effect actordefs
 	for _, actorDef := range wce.ActorDefs {
 		if actorDef.fragID > 0 {
 			continue
 		}
 
-		_, err = actorDef.ToRaw(wce, dst)
-		if err != nil {
-			return fmt.Errorf("actordef %s: %w", actorDef.Tag, err)
+		hasSpriteTag := false
+		for _, action := range actorDef.Actions {
+			for _, lod := range action.LevelOfDetails {
+				if strings.HasSuffix(lod.SpriteTag, "_SPRITE") {
+					hasSpriteTag = true
+					break
+				}
+			}
+			if hasSpriteTag {
+				break
+			}
 		}
-	}
 
-	// Write out BlitSpriteDefs
-	for _, blitSprite := range wce.BlitSpriteDefs {
-		_, err = blitSprite.ToRaw(wce, dst)
+		if hasSpriteTag {
+			continue
+		}
+
+		// fmt.Printf("Processing ActorDef: %s\n", actorDef.Tag)
```

Comment:
this comment is OK since it's handy to uncomment later when testing, so can keep <3